backend/migrations/versions/117790caec65_.py

```diff
@@ -0,0 +1,32 @@
+"""empty message
+
+Revision ID: 117790caec65
+Revises: 8b5132609f1f
+Create Date: 2023-11-16 01:53:04.353305
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '117790caec65'
+down_revision = '8b5132609f1f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('tags', schema=None) as batch_op:
+        batch_op.drop_column('status')
+
+    # ### end Alembic commands ###
+
+
+def downgrade():
```

Comment:
Running "flask db downgrade" fails and complains that the "status" column contains null values. I think it's safe to set all the tags in the db to "Active" when downgrading, since they wouldn't exist in the db if they were deleted.

Reply:
I think this is okay - any existing tags created after this is merged shouldn't have the `status` column anyways so that should solve things